doc/samtools-fixmate.1

```diff
@@ -57,7 +57,9 @@ name-sorted or name-collated alignment.
 .SH OPTIONS
 .TP 11
 .B -r
-Remove secondary and unmapped reads.
+Remove secondary and unmapped reads.  If one of a pair is removed, the PAIRED
+flag will not be unset on the remaining read.  This is a change from the older
+behaviour.
```

Comment:
Should probably add something like `in samtools versions up to 1.20`

Reply:
Done.